pom.xml

```diff
@@ -412,6 +412,11 @@
           <groupId>commons-codec</groupId>
           <artifactId>commons-codec</artifactId>
         </exclusion>
+        <!-- Excluded due to https://github.com/checkstyle/checkstyle/issues/14123 -->
```

Comment:
Please remove this comment, git commit met has it.

Reply:
i was following the pattern from the exclusion above, but OK removed the comment

Reply:
Thanks a lot.